### DISCUSSION TITLE
deb packages

### DISCUSSION THREAD

**Discussion by nodi84** [FPGAwars/icestudio#809] [Ideas]
Is there propability of creating deb package. Of corse if  anybody nead this type.

**Comment by cavearr** [comment]
Hi! this is in our roadmap for the next months. i'll tell you in this thread.

**Comment by nodi84** [comment]
Ok. I ask becouse I wrote some code in this matter and will be nice to dig deeper to obtain new knolege.

**Comment by cavearr** [comment]
Hello! I am reorganizing the issues, trying to give more visibility and be able to better organize and prioritize the work and my roadmap (I will publish it very soon), I am converting some issues that are **new feature requests** as **ideas** on Github **discussions**.

Thanks for understanding.
